src/plugins/node/loop/nodeloopplugin.cpp

```diff
@@ -68,13 +68,16 @@ static void GetLoopInformation(uv_timer_s *data) {
 #else
 static void GetLoopInformation(uv_timer_s *data, int status) {
 #endif
+    double mean = std::numeric_limits<double>::infinity();
+    if(num != 0)
+        mean = sum / num;
```

Comment:
Okay.  Can you split it off into a separate commit?  Rule of thumb: one change per commit and each with a commit log explaining what changed and why.

The bulk whitespace update you did makes the diff noisier than it needs to be but it's up to you to decide whether you want to leave it in or not.

Aside: I don't know if the style guide says yes or no on spaces before parens.